src/main/scala/scalafix/internal/sbt/ScalafixInterface.scala

```diff
@@ -31,6 +32,16 @@ object Arg {
         customDependencies.map(_.asCoursierCoordinates).asJava,
         repositories.asJava
       )
+
+    def mutableFiles: Seq[File] =
+      customURIs
```

Comment:
`customURIs` is effectively the only mutable part of `ToolClasspath` since `customDependencies` are immutable once resolved